README.md

```diff
@@ -1,16 +1,38 @@
-# gem
+![](https://img.shields.io/badge/license-MIT-007EC7.svg?style=flat-square)
+
+<div align="center">
+    <a href="http://github.com/fish-shell/oh-my-fish">
+        <img width="90" src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png">
+    </a>
+</div><br>
+
+bundler
+=======
+
 Ruby gem support for fish.
 
-## Install
-Add `gem` to your list of fish plugins in your `config.fish`.
+Install
+-------
```

Comment:
Well apparently the look and feel is the same but the code is different on the headers. But it's all good!